tests/data/crop_yield.json

```diff
@@ -433,13 +450,31 @@
         },
         {
           "name": "crop_yield__loop_plate__day_0",
-          "inputs": [
-            "consistency",
-            "max_rain",
-            "absorption",
-            "rain",
-            "total_rain",
-            "yield_est"
+          "input": [
```

Comment:
I wasn't looking for index information in the input list. I am sorry if I didn't make that clear. The objects hear should just have the name and domain information.